### PR TITLE
feat(auth-ids): Return domain with apiscope and identityresource.

### PR DESCRIPTION
**Pull request by valurefugl**
## What

Return domains with apiscope and identityresource.

## Why

So we can display them on the IDS consent page.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by eirikurn — APPROVED** [review]
Feel free to improve in a follow up.

File: libs/auth-api-lib/src/lib/resources/models/api-scope.model.ts [hunk 1]
```diff
@@ -253,6 +254,10 @@ export class ApiScope extends Model<ModelAttributes, CreationAttributes> {
   @HasMany(() => ApiScopeUserAccess)
   apiScopeUserAccesses?: ApiScopeUserAccess[]
 
+  @BelongsTo(() => Domain)
+  @ApiProperty({ type: () => Domain })
+  domain!: Domain
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply libs/auth-api-lib/src/lib/resources/models/api-scope.model.ts]: Fixed.

File: libs/auth-api-lib/src/lib/resources/models/identity-resource.model.ts [hunk 2]
```diff
@@ -154,4 +154,7 @@ export class IdentityResource extends Model {
   @UpdatedAt
   @ApiProperty()
   readonly modified?: Date
+
+  @ApiProperty({ type: () => Domain })
+  domain!: Domain
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply libs/auth-api-lib/src/lib/resources/models/identity-resource.model.ts]: Fixed.

File: libs/auth-api-lib/src/lib/resources/resources.service.ts [hunk 3]
```diff
@@ -333,10 +335,17 @@ export class ResourcesService {
...
-    return this.identityResourceModel.findAll({
+    const result = await this.identityResourceModel.findAll({
       where: scopeNames ? whereOptions : undefined,
       include: [IdentityResourceUserClaim],
     })
+
+    const identityDomain = await this.findDomainByPk(DEFAULT_DOMAIN)
+
+    return result.map((r) => {
+      r.setDataValue('domain', identityDomain)
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply libs/auth-api-lib/src/lib/resources/resources.service.ts]: Changed.

**Pull request merged**